apps/tailwind-components/components/Button.vue

```diff
@@ -55,22 +58,27 @@ const colorClasses = computed(() => {
 });
 
 const sizeClasses = computed(() => {
-  return SIZE_MAPPING[props.size];
+  return props.iconOnly ? "p-[8px]" : SIZE_MAPPING[props.size];
```

Comment:
Would it be possible create a mapping for icon only buttons? I can imagine a few instances (in the dashboards) where we want an icon only button, but would like to have a larger button. This option would make the icon only option more useful.